react/features/chat/components/AbstractMessageContainer.js

```diff
@@ -32,15 +37,25 @@ export default class AbstractMessageContainer<P: Props> extends PureComponent<P>
         const groups = [];
         let currentGrouping = [];
         let currentGroupParticipantId;
+        const localParticipant = getLocalParticipant(APP.store.getState());
 
         for (let i = 0; i < messagesCount; i++) {
             const message = this.props.messages[i];
+            const splitedMessage = message.message.split('_');
+            const akAddress = splitedMessage.length > 1 ? splitedMessage.slice(0, 2).join('_') : '';
+            const text = splitedMessage.pop();
+
+            message.akAddress = akAddress;
+            message.message = text;
+
+            if (localParticipant.name === message.displayName && message.akAddress) {
```

Comment:
i think having `&& message.akAddress` is not enough, may be also needed to check if the current user address matches the message.akAdress